notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -31,7 +31,7 @@
         "id": "fsv4jGuU89rX"
```

Comment:
<p>The links don't seem to be correct.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The links are correct, but the notebook in main is a bit older and you are seeing an old (not updated) title.</p>